### PR TITLE
Add Ansible Galaxy registry adapter to OAB docs

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@dymurray, do you have a source issue or bug for this change?

@vikram-redhat, FYI

**Comment by vikram-redhat** [comment]
@dymurray thanks. 

@kalexand-rh - this is coming out of the release themes doc for 3.11.

**Comment by vikram-redhat** [comment]
@chengzhang1016 - can you please provide QE review for this?

**Comment by chengzhang1016** [comment]
@vikram-redhat I will review it.

File: install_config/oab_broker_configuration.adoc [hunk 1]
```diff
@@ -375,6 +375,25 @@ registry:
...
+----
+registry:
+  - name: galaxy
+    type: galaxy
+    # Optional:
+    # org: ansibleplaybookbundle
+    runner: docker.io/ansibleplaybookbundle/apb-base:latest
+    white_list:
+      - ".*$"
+----
```
Comment (chengzhang1016) [inline install_config/oab_broker_configuration.adoc]: Others LGTM, one question as below.
In testing, we always using `url`, @dymurray is it a optional ? How about if without `url`?
Such as:
```
    registry:
      - type: galaxy
        name: galaxy
        org: ansibleplaybookbundle
        url: https://galaxy.ansible.com 
        runner: docker.io/ansibleplaybookbundle/apb-base:latest
        white_list: [.*]
```

Comment (chengzhang1016) [reply install_config/oab_broker_configuration.adoc]: Galaxy adapter using 'url: https://galaxy.ansible.com' as a default value and work well while without 'url' in configuration. So, this PR LGTM. Thx.

**Comment by kalexand-rh** [comment]
Thank you @chengzhang1016 and @dymurray! Merging.

**Pull request merged**